tests/src/Unit/OgAccessTestBase.php

```diff
@@ -47,6 +48,16 @@ class OgAccessTestBase extends UnitTestCase {
    */
   protected $groupManager;
 
+  /**
+   * The OgAccess class, this is the system under test.
```

Comment:
Oh that is the formal name in unit testing for the class that is being tested, it's called the "SUT" or System Under Test.

It's not yet common in the Drupalverse, I'll make it clearer.